### ISSUE TITLE
Cold start error when bootstraping a new project on Windows. 

### ISSUE THREAD

**Issue by JeanMeche** [analogjs/analog#547]
### Please provide the environment you discovered this bug in.

Running th latest node 18 (18.16.1) on Windows, creating a brand new project with `npm create analog@latest`. 

### Which area/package is the issue in?

create-analog

### Description

After runner `npm run dev`, when opening the page for the first time I get : 

```
Cannot read properties of undefined (reading 'getCurrentDirectory')
    at getRootDirs (file:///C:/Users/MatthieuRiegler/dev/ng-projects/analog-project/node_modules/@angular/compiler-cli/bundles/chunk-KPIJL3UY.js:78:20)
    at NgCompilerHost.wrap (file:///C:/Users/MatthieuRiegler/dev/ng-projects/analog-project/node_modules/@angular/compiler-cli/bundles/chunk-GD2XA2JL.js:7284:22)
    at new NgtscProgram (file:///C:/Users/MatthieuRiegler/dev/ng-projects/analog-project/node_modules/@angular/compiler-cli/bundles/chunk-GD2XA2JL.js:7362:32)
    at C:\Users\MatthieuRiegler\dev\ng-projects\analog-project\node_modules\@analogjs\vite-plugin-angular\src\lib\angular-vite-plugin.js:318:40
    at Generator.next (<anonymous>)
    at C:\Users\MatthieuRiegler\dev\ng-projects\analog-project\node_modules\tslib\tslib.js:169:75
    at new Promise (<anonymous>)
    at Object.__awaiter (C:\Users\MatthieuRiegler\dev\ng-projects\analog-project\node_modules\tslib\tslib.js:165:16)
    at buildAndAnalyze (C:\Users\MatthieuRiegler\dev\ng-projects\analog-project\node_modules\@analogjs\vite-plugin-angular\src\lib\angular-vite-plugin.js:312:24)
    at C:\Users\MatthieuRiegler\dev\ng-projects\analog-project\node_modules\@analogjs\vite-plugin-angular\src\lib\angular-vite-plugin.js:117:31
Click outside or fix the code to dismiss.
You can also disable this overlay by setting server.hmr.overlay to false in vite.config.js. 
```

This only happens on the first load but can be reproduced on every fresh start with `npm run dev`.

**Comment by brandonroberts** [comment]
Others have reported this issue also on Windows. The `host` is always created before the `NgtscProgram` is created, so not sure why it gets undefined initially on Windows.

**Comment by JeanMeche** [comment]
Do we have a way to hook up the devtools debugger when starting the SSR ? I could investigate this a bit !

**Comment by BojanKogoj** [comment]
Though I should mention, got the same error when trying Stackblitz demo (on Windows obviously).

**Comment by brandonroberts** [comment]
This issue was found by @arturovt and resolved in #558
